src/api/article.ts

```diff
@@ -60,7 +60,7 @@ export const apiArticle = {
       authHeaders(token)
     );
   },
-  async getArticleArchives(token: string, articleUUID: string, offset: number, limit: number) {
```

Comment:
nit: I was intending to implement it soon so left a few unused parameters. ESlint might complain about it but we can ignore warnings for now (it is not blocking the CI).

Reply:
Rollback this line